### DISCUSSION TITLE
Saving collection of JsonObject's

### DISCUSSION THREAD

**Discussion by ploef** [litedb-org/LiteDB#2245] [Q&A]
Hi,

Because I'm working on something that should be very generic, I'm using jsonobject. 
What should be the best way to save an collection of jsonobject's to a LiteDb collection? If at all possible.
I'm getting the following exception now:

_LiteDB.LiteException: Document has more than 20 nested documents in 'String'._

After increasing the MaxDepth to 100 the exception changed to:

 _Document has more than 100 nested documents in 'JsonValueKind'._

**Comment by varungbt** [comment]
@mbdavid: What is the best way to store a list of JSON objects and avoid this error?
